### PR TITLE
[luci] Introduce RemoveFakeQuantPass

### PR DESCRIPTION
**Pull request by seanshpark**
This will introduce RemoveFakeQuantPass that will remove all FakeQuant
Ops in the graph.

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
For #6801, from draft #6802

File: compiler/luci/pass/src/RemoveFakeQuantPass.cpp [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+#include <luci/IR/CircleNodes.h>
+
+namespace
+{
+
+bool remove_fake_quant(luci::CircleFakeQuant *fakequant)
+{
+  if (fakequant == nullptr)
+    return false;
```
Comment (jinevening) [inline compiler/luci/pass/src/RemoveFakeQuantPass.cpp:27]: This if-statement seems unnecessary. Maybe just `assert(fakequant != nullptr)`?

File: compiler/luci/pass/src/RemoveFakeQuantPass.test.cpp [hunk 2]
```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
```
Comment (jinevening) [inline compiler/luci/pass/src/RemoveFakeQuantPass.test.cpp:2]:
```suggestion
 * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
```

**Review by jinevening — DISMISSED** [review]
LGTM Thank you

File: compiler/luci/pass/src/RemoveFakeQuantPass.cpp [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+ * CircleFakeQuant OP will be removed from the output graph
+ */
+bool RemoveFakeQuantPass::run(loco::Graph *g)
+{
+  bool changed = false;
+  for (auto node : loco::active_nodes(loco::output_nodes(g)))
+  {
+    auto target_node = dynamic_cast<luci::CircleFakeQuant *>(node);
+    if (target_node != nullptr)
+      if (remove_fake_quant(target_node))
```
Comment (mhs4670go) [inline compiler/luci/pass/src/RemoveFakeQuantPass.cpp:63]: There's no case where `remove_fake_quant` returns false. How about remove `if (target_node != nullptr)` line and make it return false when `target_node == nullptr`?

Comment (seanshpark) [reply compiler/luci/pass/src/RemoveFakeQuantPass.cpp:63]: I'll revise this :) Thanks!

Comment (seanshpark) [reply compiler/luci/pass/src/RemoveFakeQuantPass.cpp:63]: Or would it be better to make `void remove_fake_quant()`....

**Review by jinevening — APPROVED** [review]
LGTM

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**